packages/blockchain/src/Blockchain.ts

```diff
@@ -43,16 +37,16 @@ export class Blockchain {
     return !!this.apiPromise.isReady;
   }
 
-  async getNextNonce() {
-    const nonce = await this.apiPromise.rpc.system.accountNextIndex<Index>(this.account.address);
+  async getNextNonce(address: string) {
```

Comment:
To make the input params closer to `accountNextIndex`

```suggestion
  async getNextNonce(address: string | AccountId) {
```